capitoli/Terzo_Passo_Variabili.rst

```diff
@@ -68,26 +75,26 @@ Le parentesi che delimitano il nome della scatola sono obbligatorie.
 
 :blue:`Qualche esercizio per memorizzare:`
      
-     -scrivi in Python questa istruzione per il calcolatore:
-      dammi la scatola GAIA3 e mettici dentro il numero 10
+     - scrivi in Python questa istruzione per il calcolatore:
+       dammi la scatola GAIA3 e mettici dentro il numero 10
 
-     -Visualizza sul calcolatore il risultato di:
-      SCATOLA2 = 10-3
-      print (SCATOLA2)
+     - Visualizza sul calcolatore il risultato di:
+       SCATOLA2 = 10-3
+       print (SCATOLA2)
 
-     -SCATOLA3 = 7*4
-      print (SCATOLA3)
+     - SCATOLA3 = 7*4
+       print (SCATOLA3)
 
-     -SCATOLA4 = 20/2*5
-      print(SCATOLA4)
+     - SCATOLA4 = 20/2*5
+       print(SCATOLA4)
 
 |
 |	**FACCIAMO UNA BREVE INTERRUZIONE.** 
```

Comment:
Perchè è così indentato?
Se vuoi fare un'interruzione, inserisci una riga verticale (come se fosse un `<hr>` in html).
Si fa con:
`-----------`
e il risultato è:
------------